sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md

```diff
@@ -0,0 +1,50 @@
+
+To utilize these new features, you must update the API version in the ARM template to *2021-01-01-preview*.
+
+**This release is currently available in the following regions; East Asia, North Europe, West Central US, East US2.** Additional regions will be added in future updates.  
+
+The following APIs are updated as part of this release:
+
+| API |
+|---------|
+| Microsoft.ServiceFabric/managedclusters |
+| Microsoft.ServiceFabric/managedclusters/nodetypes |
+| Microsoft.ServiceFabric/managedclusters/applications |
+| Microsoft.ServiceFabric/managedclusters/applicationTypes |
+| Microsoft.ServiceFabric/managedclusters/applications/services |
+
+## Features 
+
+* [ARM application deployments](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-app-deployment-template) - You can now use ARM to deploy applications to Service Fabric managed clusters. Using ARM provides greater control of resource properties, and ensures consistent resource model. With ARM application deployments, you do not have to wait for the cluster to be ready before deploying; application registration, provisioning, and deployment can all happen in one step.
+* [Disk encryption](https://docs.microsoft.com/azure/service-fabric/how-to-enable-managed-cluster-disk-encryption?tabs=azure-powershell) - Disk encryption can now be enabled on nodes in a Service Fabric managed cluster to meet compliance requirements.
+* [Automatic OS image upgrades](https://docs.microsoft.com/azure/service-fabric/how-to-managed-cluster-configuration#enable-automatic-os-image-upgrades) - Virtual Machine Scale Set automatic OS image upgrades for your Service Fabric managed cluster are now available. If a new OS version is available, the cluster node types will be upgraded one at a time. With this feature you have the option for your cluster node types to automatically be upgraded when a new OS version is available.
+* [Virtual Machine Scale Set Managed Identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-identity-managed-cluster-virtual-machine-scale-sets) - Each node type in a Service Fabric managed cluster is backed by a Virtual Machine Scale Set. To enable managed identities to be used with a managed cluster node type, a property vmManagedIdentity has been added to node type definitions. This is a list that contains user defined identities such as Azure userAssignedIdenties.  
+* [NSG rule configuration](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-configuration#networking-configurations) - You can now customize Network Security Group rules enabling you to meet any compliance requirements for your scenarios.
+
+## Known Issues
+
```

Comment:
Can we please add headers for the other feature areas and say "no known issues" ?